### PR TITLE
[TW-323] Integrations inside API Builder

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Documented adding and managing integrations on the Monitoring tab inside the API builder

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 1]
```diff
@@ -42,33 +43,71 @@ You can link monitors in your current workspace to an API version. This enables
...
+1. Under **Connect Postman to your monitoring workflows**, select a monitor integration.
+1. Enter a **Nickname** for the integration and choose a monitor. Postman will send the results of this monitor to the application you're integrating with.
+1. Finish entering the requested information. This information varies depending on the application you're integrating with, and typically includes an API key. For more help with a specific application, see [Integrating with Postman](/docs/integrations/intro-integrations/) and select [Available integrations](/docs/integrations/available-integrations/apimatic/) in the left navigation pane.
+1. Select **Add integration**.
+
+> You can configure multiple integrations for a monitor, or even have multiple instances of the same integration. For example, you can configure two Slack integrations for a monitor that send the monitor's results to two different Slack channels.
+
+### Working with monitor integrations
+
+Once you're added a monitor integration to an API version, you can take the following actions on the **Monitoring** tab:
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:96]:
```suggestion
Once you've added a monitor integration to an API version, you can take the following actions on the **Monitoring** tab:
```

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 2]
```diff
@@ -42,33 +43,71 @@ You can link monitors in your current workspace to an API version. This enables
...
+
+* Select a monitor's name to open the monitor's dashboard in a new tab.
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:98]: Maybe simplify as:

```suggestion
* Select a monitor's name to open its dashboard in a new tab.
```

❓

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md [hunk 3]
```diff
@@ -42,33 +43,71 @@ You can link monitors in your current workspace to an API version. This enables
...
+* Select **Validate** next to a monitor to validate it against the API schema (OpenAPI 3.0 schemas only). If validation is not successful, select **Issues found** and then select **Review issues**. Learn more about [validating APIs](/docs/designing-and-developing-your-api/validating-elements-against-schema/).
+* Hover over a bar in the graph to view metrics for a monitor run.
+
+  <img alt="API monitor results" src="https://assets.postman.com/postman-docs/observe-api-integrations-results-v9-10.jpg" width="332px">
+
+* Hover over the application icon for an integration to view details. Select the **Edit** icon <img alt="Edit icon" src="https://assets.postman.com/postman-docs/documentation-edit-icon-v8-10.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> to edit the integration, or select the **Delete** icon <img alt="External link icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg" width="12px" style="vertical-align:middle;margin-bottom:5px"> to remove the integration.
+
+  <img alt="Edit an API monitor" src="https://assets.postman.com/postman-docs/observe-api-integrations-edit-v9-10.jpg" width="332px">
```
Comment (HeatherStenson) [inline src/pages/docs/designing-and-developing-your-api/observing-an-api.md:106]: Up to you, but maybe obfuscate your username or use a fake one instead?

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

File: src/pages/docs/designing-and-developing-your-api/observing-an-api.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/observing-an-api.md]: Good suggestion! 🕵🏻‍♂️

**Review by claireriley — APPROVED** [review]
LGTM!

**Review by shashankawasthi88 — APPROVED** [review]
Looks good

**Pull request merged**